3.SpiralTraversal.cpp

```diff
@@ -0,0 +1,47 @@
+// Time Complexity : o(m*n)
+// Space Complexity : o(n) for storing answer
```

Comment:
You're not using any extra data structure. We don't consider the output array when calculating space complexity.